apps/spruce/src/pages/adminSettings/tabs/ExternalCommunicationsTab/ExternalCommunicationTab.tsx

```diff
@@ -0,0 +1,6 @@
+export const ExternalCommunicationsTab = () => (
+  <div className="flex flex-col gap-4">
+    <h1 className="text-xl font-bold">External Communications</h1>
```

Comment:
same comment

also, remove classNames
```suggestion
  <div>
    <h1>External Communications</h1>
```